### ISSUE TITLE
`declaration-property-value-no-unknown` breaking for a couple of valid cases

### ISSUE THREAD

**Issue by aibolit** [stylelint-scss/stylelint-scss#1118]
It seems like the rule `declaration-property-value-no-unknown` is breaking for a couple of cases that I can find with the error `✖  Cannot parse property value`

- `rgba` with a hex value
  
  ``` scss
  color: rgba(#888, 0.5);
  ```
- variables in multivalues properties
  
  ``` scss
  $foo: 1px;
  margin: 0 $foo;
  ```

**Comment by kristerkari** [comment]
Thanks for reporting!

**Comment by kevin-quinstreet** [comment]
facing the same issue.

$foo: 1px;
margin: 0 $foo; // Issue: declaration-property-value-no-unknown

**Comment by pamelalozano16** [comment]
Hi! Your package might be outdated, I tried recreating the issue and it did not raise any errors. (See [Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIAkAzCF4AICMAHAHgNwA6AdmQHQCGmwZxALpADYQBOWbMUUJ5DAWypsA5gEtSWAAyZU6PgF8yIADQgUY5jAByVAXEQx8e3FopgAzhdXgIpDSIQg6pTJmIgjDGKSgWPWADaZG5uHhYMAJ5azBIMALSQ9mIi8RFUvsJQaZb+ICGYALoqBR5sAK5aefAFLqHuIBa5APSwYMzCVAxidvG4bBC4MGxR8QBuVMzlMPGkEPHlpADWcwDu5AiYDBUwBUqkCjZJDgBi7EIMTgBWFnY2sLjWiHUNEdEwsaQJTVYBDR3eCIeEquV5RGJxRJ2BxpBgZKBZHK-TYeAEwIH5A4gBRAA)).

Regarding `rgba`, it actually doesn't accept hex values (See http://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb#values), so the warning should be raised there.

<img width="330" alt="Image" src="https://github.com/user-attachments/assets/244868e9-5055-438a-bdda-7cc1a20f46a6" />

**Comment by mr-sanders** [comment]
@pamelalozano16 rgba() accepts hex values in Sass - https://sass-lang.com/documentation/modules/#rgb

**Comment by pamelalozano16** [comment]
I totally overlooked the Sass definition for `rgba()`, thanks for pointing that out!
